cloudinit/conftest.py

```diff
@@ -0,0 +1,35 @@
+from unittest import mock
+
+import pytest
+
+
+@pytest.yield_fixture(autouse=True)
```

Comment:
I just reviewed a branch mentioning that pytest yield was not available on Xenial.  is this different?  And if so, how so?

Reply:
This change was in fact what prompted the documentation in that branch. To quote it:

> Support for using ``yield`` in ``pytest.fixture`` functions was only introduced in `pytest 3.0`_.  Such functions must instead use the ``pytest.yield_fixture`` decorator.

With modern pytest, we could (and should) write this as:

```diff
-@pytest.yield_fixture(autouse=True)
+@pytest.fixture(autouse=True)
```

but that fails on xenial.

Reply:
OK.  Thanks.